model/histogram/float_histogram.go

```diff
@@ -27,6 +27,8 @@ import (
 // used to represent a histogram with integer counts and thus serves as a more
 // generalized representation.
 type FloatHistogram struct {
+	// CounterResetHint tells the counter reset information for this histogram.
```

Comment:
IIRC the doc comments for fields don't have to start with the name of the field (see other fields in this struct). I'd prefer to have this more compact and less redundant, e.g. "// Counter reset information."